content/en/case-studies/gw-discov.md

```diff
@@ -55,81 +76,93 @@ made from warped spacetime.
 
 ## NumPy’s Role in the detection of Gravitational Waves
 	
 Gravitational waves emitted from the merger cannot be computed using any
-technique except brute force numerical relativity using supercomputers.
-The amount of data LIGO collects is as incomprehensibly large as gravitational
-wave signals are small.
-
-NumPy, the standard numerical analysis package for Python,  was utilized by
-the software used for various tasks performed during the GW detection project
-at LIGO. NumPy helped in solving complex maths and data manipulation at high
-speed.  Here are some examples:
-
-* [Signal Processing](https://www.uv.es/virgogroup/Denoising_ROF.html): Glitch
-  detection,  [Noise identification and Data Characterization](https://ep2016.europython.eu/media/conference/slides/pyhton-in-gravitational-waves-research-communities.pdf)
-  (NumPy, scikit-learn, scipy, matplotlib, pandas, pyCharm )
+technique except brute force numerical relativity using supercomputers.  The
+amount of data LIGO collects is as incomprehensibly large and requires careful
+processing like any other data science problem.  Although it is technologically
+more challenging to achieve the requisite sensitivity for capturing
+gravitaional waves and is way harder than dealing with large scale data, yet
+the latter is complex enough to require numerical processing skills of several
+Python tools including
+[NumPy](https://towardsdatascience.com/2020-how-a-i-could-help-astronomers-sorting-big-data-811571705707).
+
+NumPy, the standard numerical analysis package for Python,  was utilized by the
+software used for various tasks performed during the GW detection project at
+LIGO. NumPy helped in solving complex maths and data manipulation at high speed.
+Here are some examples:
+
+* **[Signal Processing](https://www.uv.es/virgogroup/Denoising_ROF.html):**  Glitch
+  detection,  [Noise identification and Data
+Characterization](https://ep2016.europython.eu/media/conference/slides/pyhton-in-gravitational-waves-research-communities.pdf)
+(NumPy, scikit-learn, scipy, matplotlib, pandas, pyCharm )
+* **GW data analysis:**
+  [GwPy](https://gwpy.github.io/docs/stable/overview.html) and
+[PyCBC](https://pycbc.org) are two of the [key LIGO experimentation 
+softwares](https://github.com/lscsoft) 
+that use NumPy and AstroPy under the hood, for providing
+object based interfaces to utilities, tools and methods for studying data from
+gravitational-wave detectors. Both GwPy and pycbc libraries rely on NumPy for
+analysis of the "chirp" signal. For example, in pycbc, the time series data are
+stored in ndarrays and scipy.signal, which operates on NumPy arrays.
 * Data retrieval: Deciding which data can be analyzed, figuring out whether it
   contains a signal - needle in a haystack
 * Statistical analysis: estimate the statistical significance of observational
   data, estimating the signal parameters (e.g. masses of stars, spin velocity,
```

Comment:
These bullets seem incomplete - are they notes-to-self or intended to be included? If the latter, I think they need to be developed more, though I think the above bullet about GwPy and PyCBC adequately covers components like this.